c2corg_ui/static/partials/cards/outings.html

```diff
@@ -132,7 +132,13 @@
             </div>
           </a>
         </div>
-      </div>    
+      </div>
+      <div class="parent-bubble" ng-if="doc['img_count']">
+          <span class="glyphicon glyphicon-picture" uib-tooltip="({{::doc['img_count']}})" tooltip-append-to-body="true"></span>
```

Comment:
Wouldn't a text like `3 pictures for this outing` be better than `(3)`?

Reply:
3 picture(s) is probaly enough !

Reply:
Yeah, sure. If we could handle plurals by the same occasion :pray: